centreon/src/Core/Security/Vault/Application/UseCase/FindVaultConfigurations/FindVaultConfigurationsRequest.php

```diff
@@ -21,10 +21,10 @@
 
 declare(strict_types=1);
 
-namespace Core\Security\Vault\Application\UseCase\UpdateVaultConfiguration;
+namespace Core\Security\Vault\Application\UseCase\FindVaultConfigurations;
 
-use Core\Application\Common\UseCase\PresenterInterface;
-
-interface UpdateVaultConfigurationPresenterInterface extends PresenterInterface
+final class FindVaultConfigurationsRequest
 {
+    /** @var int */
+    public int $vaultId = 0;
```

Comment:
Is it really necessary to have a Request DTO just for an id ?

Reply:
I would say yes, for consistency reasons of transferring data. But it can be disscussed.

Reply:
The DTO is not necessary in this case

Reply:
Done